standalone/include/psen_scan_v2_standalone/protocol_layer/scan_round.h

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) 2020-2021 Pilz GmbH & Co. KG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#ifndef PSEN_SCAN_V2_STANDALONE_SCAN_ROUND_H
+#define PSEN_SCAN_V2_STANDALONE_SCAN_ROUND_H
+
+#include "psen_scan_v2_standalone/data_conversion_layer/monitoring_frame_msg.h"
+#include "psen_scan_v2_standalone/util/logging.h"
+
+namespace psen_scan_v2_standalone
+{
+namespace protocol_layer
+{
+/**
+ * @brief Buffers and validates monitoring frames for a scan round.
+ *
+ * Discovers if there are to many monitoring frames in a scan round.
+ * Informs when a scan round ended incomplete.
+ * Discovers and omits old messages.
+ */
+class ScanRound
```

Comment:
> Buffers and validates monitoring frames for a scan round.

I suggest something like `ScanBuffer`. I am not even sure if "Scan Round" is a proper english word.